test/JDBC/input/BABEL-1251-vu-verify.sql

```diff
@@ -0,0 +1,58 @@
+SET IDENTITY_INSERT babel_1251_vu_prepare.t1 ON;
+go
+INSERT INTO babel_1251_vu_prepare.t1(col1, id) VALUES (1, 10);
```

Comment:
prepare will make more sense, why in verify?

Reply:
We are using testing for the @@Identity values after the INSERT statement ,If insert statements are in prepare we cant check wether identity values are working properly or not

Reply:
we can break it where we keep few inserts in prepare and others in verify

Reply:
I have moved some inserts in prepare and have only the required inserts in verify script in the latest commit